sigstore-scaffolding.yaml

```diff
@@ -1,14 +1,18 @@
 package:
   name: sigstore-scaffolding
   version: 0.6.8
-  epoch: 1
+  epoch: 2
   description: Software Supply Chain Transparency Log
   target-architecture:
     - all
   copyright:
     - license: Apache-2.0
       paths:
         - "*"
+  dependencies:
+    runtime:
+      - cloud-sql-proxy
+      - cloud-sql-proxy-compat
```

Comment:
This will add it to the meta package, where we want it on the `cloudsqlproxy` subpackage.

For this, I'd probably split `cloudsqlproxy` out of the "range" below and just create a dedicated copy for it where we can add the dependency.

Reply:
done, ptal